components/root/root.css

```diff
@@ -71,11 +71,22 @@ button{
     border: none;
 }
 
-
+.spacing-dividing{
+    margin: 5rem;
+}
 /* General utility classes */
 .circle-border-radius{
     border-radius: 50%;
 }
 .smooth-square-radius{
     border-radius: 4px;
+}
+.sub-text{
+    font-size:1.6rem;
+    color: var(--6C757D);
```

Comment:
Keep naming consistent, some where you have given a name like `PURE-BLACK`, and some where you are using color code for naming.

Reply:
I have used it as in sense that the colors which are mostly used have name else they are used by color codes